### DISCUSSION TITLE
Contributions

### DISCUSSION THREAD

**Discussion by Marin-Kitagawa** [anuraghazra/github-readme-stats#1129] [Q&A] [answered]
What does  "Contributed to" exactly mean here?

Does it mean any of the following?
1. Contribution to "other's repositories" via a pull and merge
2. If I have "Contributed to : `n`", does that mean I have contributed to any `n` repositories?
3. Any of [this](https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/managing-contribution-graphs-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution) said as a contribution by Github

**Comment by florianbussmann** [comment, marked as answer]
I already answered this question here https://github.com/anuraghazra/github-readme-stats/issues/1125#issuecomment-856646890
It means you have contributed to `n` repositories, but only commits, issues and pull requests will be considered as contribution.

**Reply by davidshare** [reply]
@florianbussmann - I have several contributions. On the teams I have worked with, I have more than 10 repositories, my personal repositories are over 20. But I see contributed to at 10. Please how is this calculated?

**Reply by florianbussmann** [reply]
@davidshare below you will find the 10 repositories listed that were counted as contribution so far. Repositories you own will not be considered for the calculation. Are there more repos you contributed to?

1. fluent/fluentd
2. uken/fluent-plugin-elasticsearch
3. silinternational/ecs-deploy
4. vmware-tanzu/velero
5. etcd-io/discovery.etcd.io
6. material-shell/material-shell
7. cloudkite-io/discovery.etcd.io
8. cloudkite-io/terraform-modules
9. WyriHaximus/github-action-helm3
10. AGs-hue/firstproject
